templates/openstack/index.html

```diff
@@ -769,10 +769,27 @@ <h2 class="p-heading--4">
 </section>
 
 <section class="p-strip--light">
-  <div class="u-fixed-width">
+  <div class="u-hide--large u-fixed-width">
     <h2>Number 1 platform for OpenStack implementation</h2>
-    <p>According to the <a class="p-link--external" href="https://www.openstack.org/analytics/">OpenStack User Survey 2020</a> results, Ubuntu Server is the most popular operating system for OpenStack implementation. Ubuntu Server powers 40% of OpenStack clouds all over the world. It has been chosen as a platform for private cloud implementation by leading companies in the telco, finance, hardware manufacturing, retail, automotive and healthcare sectors.</p>
   </div>
+  <div class="row">
+    <div class="u-hide--small u-hide--medium col-6">
+      <h2>Number 1 platform for OpenStack implementation</h2>
```

Comment:
I tested different approach on local to avoid the duplicated content just for the sake of responsiveness, but couldn't find a better result, well done :+1:

Reply:
Oh I just followed how Pete had done it, can't take credit for that haha. Thanks for the review!